Library/Homebrew/test/cleanup_spec.rb

```diff
@@ -58,13 +52,13 @@
       end
 
       it "doesn't remove any kegs" do
-        shutup { described_class.cleanup_formula f2 }
+        described_class.cleanup_formula f2
         expect(f1.installed_kegs.size).to eq(2)
       end
 
       it "lists the unremovable kegs" do
         shutup { described_class.cleanup_formula f2 }
-        expect(described_class.unremovable_kegs).to contain_exactly(f1.installed_kegs.sort_by(&:to_s)[0])
+        expect(described_class.unremovable_kegs).to contain_exactly(f1.installed_kegs[0])
```

Comment:
This test may fail due to the file system sorting either chronologically or lexicographically. Try
```ruby
expect(described_class.unremovable_kegs).to contain_exactly(f1.installed_kegs.sort_by(&:to_s)[0])
```